xml-web-services/2.3/jakarta.xml.ws-spec-2.3.html

```diff
@@ -0,0 +1,661 @@
+.CodeRay .shell{color:#d14}
+.CodeRay .shell .delimiter{color:#d14}
+.CodeRay .symbol{color:#990073}
+.CodeRay .symbol .content{color:#a60}
+.CodeRay .symbol .delimiter{color:#630}
+.CodeRay .tag{color:#008080}
+.CodeRay .tag-special{color:#d70}
+.CodeRay .variable{color:#036}
+.CodeRay .insert{background:#afa}
+.CodeRay .delete{background:#faa}
+.CodeRay .change{color:#aaf;background:#007}
+.CodeRay .head{color:#f8f;background:#505}
+.CodeRay .insert .insert{color:#080}
+.CodeRay .delete .delete{color:#800}
+.CodeRay .change .change{color:#66f}
+.CodeRay .head .head{color:#f4f}
+</style>
+</head>
+<body class="book toc2 toc-left">
+<div id="header">
+<h1>Jakarta XML Web Services</h1>
+<div class="details">
+<span id="author" class="author">Jakarta XML Web Services Team, https://projects.eclipse.org/projects/ee4j.jaxws</span><br>
+<span id="email" class="email"><a href="https://dev.eclipse.org/mailman/listinfo/jaxws-dev" class="bare">https://dev.eclipse.org/mailman/listinfo/jaxws-dev</a></span><br>
+<span id="revnumber"> 2.3,</span>
+<span id="revdate">July 22, 2019</span>
+<br><span id="revremark">Final Release</span>
+</div>
+<div id="toc" class="toc2">
+<div id="toctitle">Table of Contents</div>
+<ul class="sectlevel2">
+<li><a href="#eclipse-foundation-specification-license">Eclipse Foundation Specification License</a>
+<ul class="sectlevel3">
+<li><a href="#disclaimers">Disclaimers</a></li>
+</ul>
+</li>
+<li><a href="#introduction">1. Introduction</a></li>
+</ul>
+</div>
+</div>
+<div id="content">
+<div id="preamble">
+<div class="sectionbody">
+<div class="literalblock">
+<div class="content">
+<pre>Specification: Jakarta XML Web Services
+
+Version: 2.3
+
+Status: Final Release
+
+Release: July 22, 2019</pre>
+</div>
+</div>
+</div>
+</div>
+<div class="sect2">
+<h3 id="eclipse-foundation-specification-license"><a class="anchor" href="#eclipse-foundation-specification-license"></a>Eclipse Foundation Specification License</h3>
+<div class="paragraph">
+<p>By using and/or copying this document, or the Eclipse Foundation
+document from which this statement is linked, you (the licensee) agree
+that you have read, understood, and will comply with the following
+terms and conditions:</p>
+</div>
+<div class="paragraph">
+<p>Permission to copy, and distribute the contents of this document, or
+the Eclipse Foundation document from which this statement is linked, in
+any medium for any purpose and without fee or royalty is hereby
+granted, provided that you include the following on ALL copies of the
+document, or portions thereof, that you use:</p>
+</div>
+<div class="ulist">
+<ul>
+<li>
+<p>link or URL to the original Eclipse Foundation document.</p>
+</li>
+<li>
+<p>All existing copyright notices, or if one does not exist, a notice
+(hypertext is preferred, but a textual representation is permitted)
+of the form: "Copyright (c) [$date-of-document]
+Eclipse Foundation, Inc. <a href="#url to this license">[url to this license]</a>"</p>
```

Comment:
When I pull this PR and opened the .html via IntelliJ preview view, this line renders like the attached email.
Is this intended so the Eclipse website provides the placeholder or we should have actually a link to a license?

![image](https://user-images.githubusercontent.com/1454523/62304082-868cfb80-b43a-11e9-98b2-b7699038206e.png)

Reply:
You get the right/full link when you either right-click on it and choose copy or click on the link and check address bar in the browser

Reply:
I think Cesar mentions the `<a href="#url to this license">[url to this license]</a>` placeholder. Also, the [$date-of-document] probable needs to be replaced with some value.

Reply:
Ok. What one gets after clicking on the [Url to this license] link? Isn’t it the actual url to the license?

Reply:
This part of the license is instructions for how you're supposed to update the document if you make a derivative copy of the document.  When you make a copy you're supposed to fill in
that part with the actual URL to the license.